Src/Support/Google.Apis.Auth/OAuth2/GoogleCredential.cs

```diff
@@ -326,6 +326,45 @@ void IConfigurableHttpClientInitializer.Initialize(ConfigurableHttpClient httpCl
         /// </summary>
         public ICredential UnderlyingCredential => credential;
 
+        /// <summary>Creates a <c>GoogleCredential</c> wrapping a <see cref="ImpersonatedCredential"/>.</summary>
```

Comment:
Use <see cref="GoogleCredential" /> instead of `<c>` here. (You may well have been following a pattern, but we should fix the others rather than propagating this :)

Reply:
done